src/marqo/upgrades/v20_v21.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from marqo import logging, version
+from marqo.api import exceptions as api_exceptions
+from marqo.core.index_management.index_management import IndexManagement, _MarqoConfig
+from marqo.upgrades.upgrade import Upgrade
+from marqo.vespa.models import VespaDocument
+from marqo.vespa.vespa_client import VespaClient
+
+logger = logging.get_logger(__name__)
+
+
+class v20v21(Upgrade):
+    """
+    Upgrade from Marqo v2.0 to v2.1.
+
+    The migration comprises two steps:
+    1. Create the default query profile
+    2. Add Marqo version to Marqo settings schema
+    """
```

Comment:
does this need a validation step? checking if the current version _is_ v2.0?

Reply:
`UpgradeRunner` does this validation and won't call this class if versions aren't right.